### PR TITLE
Fix image print for auto feedback from code notebook

### PR DESCRIPTION
**Pull request by pmalarme**
## Why are these changes needed?

It solves the display of the chart in notebook/agentchat_auto_feedback_from_code_execution.ipynb. I also added .notebook in the `.gitignore` to be able to have a folder to update the notebooks for testing without risking to commit them.

## Checks

- [x] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [x] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [x] I've made sure all auto checks have passed.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 .docusaurus/
+.notebook/
```
Comment (randombet) [inline .gitignore:2]: IMAO, as long as this is part of the project, it should not be added here. 
Files listed in .gitignore are usually temporary files, sensitive data, or local files.

File: notebook/agentchat_auto_feedback_from_code_execution.ipynb [hunk 2]
```diff
@@ -813,7 +814,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.11.4"
+   "version": "3.10.13"
```
Comment (randombet) [inline notebook/agentchat_auto_feedback_from_code_execution.ipynb:817]: What is the specific reason to change it here? What was wrong with 3.11.4?

Comment (davorrunje) [reply notebook/agentchat_auto_feedback_from_code_execution.ipynb:817]: I believe this is added automatically as metadata when you execute the notebook. There are scripts to strip such metadata, typically invoked by pre-commit (see e.g. https://nbdev.fast.ai/api/clean.html)

**Comment by davorrunje** [comment]
Let me just understand the problem before I get on my laptop and run it myself, the issue was that the image was not displayed in notebook?

If so, wouldn't it be better to check if the file exists, load it from file and then display it? That way we know what is in the file is actually what we are looking into. Not saying the PR is wrong, but this solution would be more robust.

File: notebook/agentchat_auto_feedback_from_code_execution.ipynb (hunk 2 above)
Comment (pmalarme) [reply notebook/agentchat_auto_feedback_from_code_execution.ipynb:817]: Yes it seems to have been added automatically

**Comment by pmalarme** [comment]
> Let me just understand the problem before I get on my laptop and run it myself, the issue was that the image was not displayed in notebook?
> 
> If so, wouldn't it be better to check if the file exists, load it from file and then display it? That way we know what is in the file is actually what we are looking into. Not saying the PR is wrong, but this solution would be more robust.

The image do not display in GitHub Codespace. You just ask explicitly to display it and now it works.

File: .gitignore (hunk 1 above)
Comment (davorrunje) [reply .gitignore:2]: You might be right about it being useful, but that should not be a part of this PR.

Comment (pmalarme) [reply .gitignore:2]: I will update it

**Review by davorrunje — APPROVED** [review]

**Comment by davorrunje** [comment]
LGTM 👍

**Pull request merged**